nengo/spa/tests/test_basalganglia.py

```diff
@@ -22,6 +22,7 @@ def test_basal_ganglia(Simulator, seed, plt):
             'dot(vision,CAT)+0.5-dot(vision,CAT) --> motor=E',
             'dot(vision,PARROT) + compare --> motor=F',
             '0.5*dot(vision,MOUSE) + 0.5*compare --> motor=G',
+            '( dot(vision,MOUSE) - compare) * 0.5 --> motor=H'
```

Comment:
But I would be fine with merging the PR as is (there might be more important things than the formatting and most of it was that way already before).